neurodsp/tests/test_spectral_power.py

```diff
@@ -40,6 +44,26 @@ def test_compute_spectrum_welch(tsig):
     freqs, spectrum = compute_spectrum_welch(tsig, FS, avg_type='median')
     assert freqs.shape == spectrum.shape
 
+    # Create a sinusoid with 10 periods/cycles.
+    n_seconds = 10
+    fs = 100
+    sig = sim_oscillation(n_seconds, fs, 1, n_cycles=n_seconds, mean=None, variance=None)
```

Comment:
The `n_cycles` input isn't needed / doesn't do anything here (at least I don't think it does), and shouldn't be needed.

Reply:
Yup, good call!